packages/eslint-config-udemy-basics/parts/stylistic-issues.js

```diff
@@ -10,7 +10,10 @@ module.exports = {
         // enforce one true brace style
         'brace-style': ['error', '1tbs', { allowSingleLine: true }],
         // require camel case names
-        camelcase: ['error', { properties: 'never' }],
+        camelcase: ['error', {
+            properties: 'never',
+            allow: ['UNSAFE_componentWillMount', 'UNSAFE_componentWillReceiveProps', 'UNSAFE_componentWillUpdate'],
```

Comment:
Let's definitely create a follow-up task to get rid of these UNSAFE_ usages and delete this configuration eventually.

Reply:
Sure. Do you think it would be better to just copy this configuration over to the `.eslintrc` file in the website-django codebase and delete it once we fully deprecate those methods?

Reply:
I'm okay either way. Since you're doing Team F work, feel free to create a card on our board as Cansin asked.